### ISSUE TITLE
Tag new version?

### ISSUE THREAD

**Issue by petrkotek** [paypal/sdk-core-php#90]
I believe it would be handy to tag new version for two reasons:
- unit tests are failing under `v3.2.4`, but succeeding under `master` (see output below)
- `master` now runs tests automatically (travis, see #88), which (sort of) declares compatibility with PHP7 (which could be important for many people; btw, would be nice to mention the support explicitely in the `README.me`!)

### Required Information

- PHP Version: 7.0.10
- SDK-Core-PHP Version: v3.2.4
- Debug ID(s): n/a

### Issue Description
Unit tests failing under v3.2.4:
```
$ vendor/bin/phpunit
PHPUnit 3.7.38 by Sebastian Bergmann.

The Xdebug extension is not loaded. No code coverage will be generated.


PHPUnit 3.7.38 by Sebastian Bergmann.

Configuration read from /(...)/sdk-core-php/phpunit.xml

...............................................................  63 / 134 ( 47%)
....................................................F.F........ 126 / 134 ( 94%)
..F.....

Time: 51.65 seconds, Memory: 6.00MB

There were 3 failures:

1) PPGenericServiceHandlerTest::testHeadersAdded
Failed asserting that 6 matches expected 5.

/(...)/sdk-core-php/tests/handlers/PPGenericServiceHandlerTest.php:33

2) PPMerchantServiceHandlerTest::testHeadersAdded
Basic headers not added
Failed asserting that 5 matches expected 4.

/(...)/sdk-core-php/tests/handlers/PPMerchantServiceHandlerTest.php:46

3) PPPlatformServiceHandlerTest::testHeadersAdded
Basic headers not added
Failed asserting that 9 matches expected 8.

/(...)/sdk-core-php/tests/handlers/PPPlatformServiceHandlerTest.php:66

FAILURES!
Tests: 134, Assertions: 314, Failures: 3.
```

However tests are succeeding under `master`:
```
$ vendor/bin/phpunit
PHPUnit 3.7.38 by Sebastian Bergmann.

The Xdebug extension is not loaded. No code coverage will be generated.


PHPUnit 3.7.38 by Sebastian Bergmann.

Configuration read from /(...)/sdk-core-php/phpunit.xml

...............................................................  63 / 132 ( 47%)
............................................................... 126 / 132 ( 95%)
......

Time: 47.82 seconds, Memory: 6.00MB

OK (132 tests, 317 assertions)
```

**Comment by jaypatel512** [comment]
Released the new version [3.3.1](https://github.com/paypal/sdk-core-php/releases/tag/3.3.1), as we had few other bug fixes to be released along with your travis updates too 👍 .

Thanks for helping us out here. Appreciated.

**Comment by petrkotek** [comment]
thanks for your help, that's fantastic!
